### PR TITLE
Order page revamp

### PR DESCRIPTION
**Pull request by Soontobecoder**
changed the layout of order page to show your cart on the left

### CODE REVIEW

File: client/src/pages/order/data/data.ts [hunk 1]
```diff
@@ -1,68 +1,74 @@
...
 ];
 
 const MODELS = [
   {
-    value: '1',
-    label: 'KELANA - Waroeng Cak Timmies Hoodie',
-    price: 32,
+    value: "1",
+    label: "KELANA - Waroeng Cak Timmies Hoodie",
+    price: 35,
```
Comment (Julian-UBC) [inline client/src/pages/order/data/data.ts:51]: 32

File: client/src/pages/order/data/data.ts [hunk 2]
```diff
@@ -1,68 +1,74 @@
...
+    image:
+      "https://permikawebsite.s3.us-west-2.amazonaws.com/assets/merchandise/hoodieBlackFront.webp",
   },
   {
-    value: '2',
-    label: 'KELANA - Anak Rantau Hoodie',
-    price: 32,
+    value: "2",
+    label: "KELANA - Anak Rantau Hoodie",
+    price: 35,
```
Comment (Julian-UBC) [inline client/src/pages/order/data/data.ts:58]: 32

File: client/src/pages/order/data/data.ts [hunk 3]
```diff
@@ -1,68 +1,74 @@
...
+    image:
+      "https://permikawebsite.s3.us-west-2.amazonaws.com/assets/merchandise/hoodieWhiteFront.webp",
   },
   {
-    value: '3',
-    label: 'KELANA - Anak Rantau T-Shirt',
-    price: 18,
+    value: "3",
+    label: "KELANA - Anak Rantau T-Shirt",
+    price: 20,
```
Comment (Julian-UBC) [inline client/src/pages/order/data/data.ts:65]: 18

**Review by Julian-UBC — CHANGES_REQUESTED** [review]

File: client/src/pages/order/Order.tsx [hunk 4]
```diff
@@ -457,362 +462,403 @@ export const Order = () => {
...
+                      </h2>
+                      <p className="payment-description pt-[2%] text-grey-body">
+                        Please access the link below and pay your total amount
+                        there.
+                      </p>
+                      <p
+                        onClick={() => {
+                          setPaymentClicked(true);
+                          openExternalLink(
+                            'https://forms.gle/wSxUEACu6ydqegVh9'
```
Comment (Julian-UBC) [inline client/src/pages/order/Order.tsx:807]:
```suggestion
                            'https://forms.gle/RbuP9j3wqG5ZKAuaA'
```

File: client/src/pages/order/Order.tsx [hunk 5]
```diff
@@ -457,362 +462,403 @@ export const Order = () => {
...
+                      </p>
+                    </div>
+                  </div>
+                )}
+                {!isTotalsVisible && shoppingBag.length === 0 && <div></div>}
+                {page === 'payment' ? (
+                  <button
+                    className="bg-[#D07D14] w-full rounded-md text-white py-1.5 text-lg mt-7 disabled:bg-gray-400"
                     onClick={() => {
                       setPaymentClicked(true);
```
Comment (Julian-UBC) [inline client/src/pages/order/Order.tsx:822]:
```suggestion
                      setSubmitOrderClicked(true);
```

File: client/src/pages/order/Order.tsx [hunk 6]
```diff
@@ -457,362 +462,403 @@ export const Order = () => {
...
                       openExternalLink('https://forms.gle/RbuP9j3wqG5ZKAuaA');
```
Comment (Julian-UBC) [inline client/src/pages/order/Order.tsx:823]:
```suggestion
                      handleNextPage();
```

**Review by Julian-UBC — CHANGES_REQUESTED** [review]

**Review by Julian-UBC — APPROVED** [review]

**Review by rliebig2004 — APPROVED** [review]
looks good!

**Pull request merged**